manual/sphinx/user_docs/time_integration.rst

```diff
@@ -25,69 +25,75 @@ or on the command line by adding ``solver:type=pvode`` for example:
+   | karniadakis   | Karniadakis explicit method             | Always available   |
+   +---------------+-----------------------------------------+--------------------+
+   | pvode         | 1998 PVODE with BDF method              | Always available   |
+   +---------------+-----------------------------------------+--------------------+
+   | cvode         | SUNDIALS CVODE. BDF and Adams methods   | –with-cvode        |
+   +---------------+-----------------------------------------+--------------------+
+   | ida           | SUNDIALS IDA. DAE solver                | –with-ida          |
+   +---------------+-----------------------------------------+--------------------+
+   | petsc         | PETSc TS methods                        | –with-petsc        |
+   +---------------+-----------------------------------------+--------------------+
+   | imexbdf2      | IMEX-BDF2 scheme                        | –with-petsc        |
+   +---------------+-----------------------------------------+--------------------+
+
+|
 
 Each solver can have its own settings which work in slightly different
 ways, but some common settings and which solvers they are used in are
-given in table [tab:solveropts].
-
-+------------------+--------------------------------------------+-------------------------------------+
-| Option           | Description                                | Solvers used                        |
-+==================+============================================+=====================================+
-| atol             | Absolute tolerance                         | rk4, pvode, cvode, ida, imexbdf2    |
-+------------------+--------------------------------------------+-------------------------------------+
-| rtol             | Relative tolerance                         | rk4, pvode, cvode, ida, imexbdf2    |
-+------------------+--------------------------------------------+-------------------------------------+
-| mxstep           | Maximum internal steps                     | rk4, imexbdf2                       |
-|                  | per output step                            |                                     |
-+------------------+--------------------------------------------+-------------------------------------+
-| max\_timestep    | Maximum timestep                           | rk4, cvode                          |
-+------------------+--------------------------------------------+-------------------------------------+
-| timestep         | Starting timestep                          | rk4, karniadakis, euler, imexbdf2   |
-+------------------+--------------------------------------------+-------------------------------------+
-| adaptive         | Adapt timestep? (Y/N)                      | rk4, imexbdf2                       |
-+------------------+--------------------------------------------+-------------------------------------+
-| use\_precon      | Use a preconditioner? (Y/N)                | pvode, cvode, ida, imexbdf2         |
```

Comment:
Is it necessary to escape these underscores?

Reply:
Probably not!